src/routes/guides/testing-your-app.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Testing your App
+order: 6
+---
+
+This guide should cover the basics of testing a Solid.js application to inspire confidence during the further development.
```

Comment:
Final comment from my side:
In the docs, we try to use just Solid or SolidJS. Can we replace all instances of Solid.js here for one of those?
Also can we make sure we have no lowercase solid as well :)

Reply:
Can do.